components/support/migration/pings.yaml

```diff
@@ -0,0 +1,16 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+$schema: moz://mozilla.org/schemas/glean/pings/1-0-0
+
+migration:
+  description: >
+    A ping sent after a Fennec->Fenix migration was completed.
+  include_client_id: true
+  bugs:
+  - https://github.com/mozilla-mobile/fenix/issues/5888
+  notification_emails:
+  - fenix-core@mozilla.com
+  data_reviews:
+  - https://github.com/mozilla-mobile/android-components/pull/5483
```

Comment:
Please do not forget to update this to the response of the data review.

Reply:
Okay, will do.

We haven't followed this pattern in at least some other metrics that were added (e.g. a lot just link to the PR with a data review), and this isn't mentioned in the glean docs, either.

Reply:
> We haven't followed this pattern in at least some other metrics that were added (e.g. a lot just link to the PR with a data review), and this isn't mentioned in the glean docs, either.

Unfortunately this is something we cannot enforce at build-time and we have to resort to documentation and education.

I created a PR to [clarify the language in our docs](https://github.com/mozilla/glean/pull/641).